### PR TITLE
added a banner to warn the player if the worm cap is full

### PR DESCRIPTION
**Pull request by Vahvl**
i hope this works fine lmao, seemed to be okay during testing, the only thing i forgor to test was how 7.2 size worked

### CODE REVIEW

**Comment by j10a1n15** [comment]
image :)

**Comment by Vahvl** [comment]
i have been informed that the size of the banner might go off screen so time to test

**Comment by Vahvl** [comment]
nonono why did the testing one get pushed here

**Comment by Vahvl** [comment]
github moment

**Comment by Vahvl** [comment]
the size is very much fine
image

File: src/main/java/at/hannibal2/skyhanni/features/fishing/FishingTimer.kt [hunk 1]
```diff
@@ -67,6 +69,7 @@ class FishingTimer {
 
         if (inHollows && newCount >= 60 && config.wormLimitAlert) {
             SoundUtils.playBeepSound()
+            TitleManager.sendTitle("§cWORM CAP FULL!!!", 2.seconds, 7.2)
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/fishing/FishingTimer.kt]: Please change this to LorenzUtils.sendTitle()
if you want you can also change the other two instances of TitleManager.sendTitle() in the codebase to the LorenzUtils one

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
Everything looks fine apart from this

**Comment by Vahvl** [comment]
i think that works, cal tell me if anything is wrong

**Review by CalMWolfs — APPROVED** [review]

**Comment by hannibal002** [comment]
image
please create a new branch for your next pr

**Review by hannibal002 — APPROVED** [review]

**Comment by Vahvl** [comment]
alright. but why is creating a new branch important?

**Pull request merged**